src/Adapter/Order/AbstractOrderHandler.php

```diff
@@ -66,4 +66,16 @@ protected function isTaxIncludedInOrder(Order $order): bool
 
         return $taxCalculationMethod === PS_TAX_INC;
     }
+
+    /**
+     * @param Order $order
+     *
+     * @return int
+     */
+    protected function getOrderTaxCalculationMethod(Order $order): int
+    {
+        $customer = new Customer($order->id_customer);
+
+        return Group::getPriceDisplayMethod((int) $customer->id_default_group);
```

Comment:
I like the idea of having this method in the abstract class
Could you add another one to detect if tax is included? Which is often what we really want to know
```php
    /**
     * @param Order $order
     *
     * @return bool
     */
    protected function isTaxIncludedInOrder(Order $order): bool
    {
        return $this-> getOrderTaxCalculationMethod($order) === PS_TAX_INC;
    }
```

Reply:
Its there already as I see, git says you've already added it at 2020-01-09, but its not being used anywhere tho :smile:

Reply:
Damn you're right 😅 I guess many code has been refacto and thus this function is not used any more